omod/src/main/resources/config.xml

```diff
@@ -1351,7 +1351,16 @@
 		<property>@MODULE_ID@.masterCardDrugPickupEncounterTypeUuid</property>
 		<defaultValue>cce7b2ec-4fc0-4f9e-806d-b1c4ee9577db</defaultValue>
 		<description>
-			UUID for the Levantamento de ARV Master Card encounter type
+            UUID for the Levantamento de ARV Master Card encounter type
+		</description>
+	</globalProperty>
+
+	<!-- 51 -->
+	<globalProperty>
+		<property>eptsreports.fsrEncounterType</property>
+		<defaultValue>e2790f68-1d5f-11e0-b929-000c29ad1d07</defaultValue>

```

Comment:
What is the reason for mapping to same uuid as `MISAU: LABORATORIO`?

Reply:
I used that as a place holder since NOT all implementations have that encounter type present